src/Uno.UI/UI/Xaml/Media/AcrylicBrush/Android/AndroidStockBlurImpl.Android.cs

```diff
@@ -0,0 +1,120 @@
+﻿//------------------------------------------------------------------------------
```

Comment:
Let's mention this in `THIRD-PARTY-NOTICES.md`

Reply:
Did you make modifications to it ?

Reply:
Just some refactoring, will add the mentions